pkg/solana/example_test.go

```diff
@@ -0,0 +1,46 @@
+package solana_test
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/smartcontractkit/chainlink-common/pkg/codec"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestExample(t *testing.T) {
```

Comment:
nit: can we name this something more specific?

Reply:
This file should be removed. It doesn't do anything useful for test coverage.